app/controllers/admin/rdv_wizard_steps_controller.rb

```diff
@@ -15,6 +15,11 @@ def new
     @rdv = @rdv_wizard.rdv
     set_services_and_motifs if current_step == "step1"
     authorize(@rdv_wizard.rdv, :new?, policy_class: Agent::RdvPolicy)
+
+    @rdv_wizard.rdv.agents.each do |agent|
+      Caldav::SyncAbsencesJob.perform_later(agent.id) if @agent.caldav_configured?
```

Comment:
c'est la variable locale plutôt que la variable d'instance ici je crois
```suggestion
      Caldav::SyncAbsencesJob.perform_later(agent.id) if agent.caldav_configured?
```

Reply:
j'avoue que ça me donne envie qu'on ajoute un tout petit peu de tests sur cette feature. Ça pourrait être juste quelque chose de basique qui teste qu'on enqueue le bon job quand on commence une prise de rendez-vous pour un agent avec une synchro caldav

Reply:
ici on lance une synchro caldav pour les agents pour lesquels on démarre une prise de rendez-vous, mais c'est après qu'on ai fait la recherche de créneaux. Je serais assez tenté de plutôt déplacer ça dans `CreneauxSearch::Calculator`, puisqu'il est déjà appelé automatiquement quand on cherche les créneaux.
On pourrait aussi l'appeler de manière "préventive" quand quelqu'un charge la page de prise de rdv d'une orga et/ou la page de recherche de créneaux côté agents.

Reply:
@victormours avant que j’écrive les tests, tu verrais quelque chose comme ça ? https://github.com/betagouv/rdv-service-public/pull/5697/commits/19d038dddaed6047a40332957f0f64272fe3ff5a

Reply:
yes !